lib/eventq/eventq_base/queue_message.rb

```diff
@@ -11,6 +11,7 @@ class QueueMessage
     attr_accessor_type :signature, type: String
     attr_accessor_type :context, type: Hash
     attr_accessor_type :correlation_trace_id, type: String
```

Comment:
```suggestion
```

Then we could add this to maintain the same API:

```ruby
def correlation_trace_id
  Correlation['Trace']
end
```

Reply:
Unfortunately this won't work as it as it won't get parsed when the message is serialized as JSON.  Ideally I would like to remove `correlation_trace_id` but it would be a breaking change.